### PR TITLE
tests: add TypeError tests

### LINKED ISSUE #5: Add test for importing BigInt where it's not expected
As Dan's review pointed out https://github.com/WebAssembly/JS-BigInt-integration/pull/4#pullrequestreview-167513149.

### PR DESCRIPTION
**Pull request by xtuc**
Fix #5 

The tests in params and exports are a bit overlapping but otherwise we don't cover all the cases.

### CODE REVIEW

File: test/js-api/module/params.any.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          .addBody()
+          .exportFunc();
+
+      return builder.toBuffer();
+  })();
+
+  const module = new WebAssembly.Module(exportingModuleLongIdentityFn);
+  const instance = new WebAssembly.Instance(module);
+
+  instance.exports.id(123n);
```
Comment (littledan) [inline test/js-api/module/params.any.js:20]: Can you also test the behavior of this function when it's passed a float argument?

**Review by littledan — APPROVED** [review]
Nit: id is a sort of strange name for a function returning void; probably best to either rename it or make it return the argument.

Nit: The name "params" is a little funny when the point of the test is purely to check long/BigInt params; maybe name it long-params.any.js.

**Comment by xtuc** [comment]
- `id` was the short for identity, so I will change the name.

I'm still working on the tests in v8 (because I can actually run them), i'll port them here or upgrade the wasm builder.

**Comment by xtuc** [comment]
Closing for #8 (will be updated soon)

**Pull request closed without merging**